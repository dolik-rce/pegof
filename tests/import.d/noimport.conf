input import.d/import.peg
import import.d/dir2
optimize inline,remove-group,concat-strings
no-follow
header never
