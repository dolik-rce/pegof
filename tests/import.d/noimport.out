main <- A B C D E "f"

%import "import1.peg"

%import "dir1/import2.peg"

%import "import3.peg"

%import "import4.peg"

%import "import5.peg"
